### DISCUSSION TITLE
What if i want to change "type of library do you want to develop" later is that possible

### DISCUSSION THREAD

**Discussion by Chandravardhan-Singh** [callstack/react-native-builder-bob#392] [Q&A] [answered]
What if want to change "type of library do you want to develop" later is that possible

**Comment by atlj** [comment, marked as answer]
The option you chose for the library type affects which template is used to instantiate your project. It would be nearly impossible to change it later on automatically since there are a lot of differences between different templates.

So if you are in a situation where you need to change the library type later on, you should create a new project and move the code manually.
